_episodes_rmd/04-raster-calculations-in-r.Rmd

```diff
@@ -277,110 +277,110 @@ writeRaster(CHM_ov_HARV, "chm_HARV.tiff",
+> collected in Southern California. The
+> <a href="http://www.neonscience.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >NEON San Joaquin Experimental Range (SJER) field site </a>
 > located in Southern California has a very different ecosystem and climate than
 > the
 > <a href="http://www.neonscience.org/science-design/field-sites/harvard-forest" target="_blank" >NEON Harvard Forest Field Site</a>
-in Massachusetts.  
-> 
+in Massachusetts.
+>
 > Import the SJER DSM and DTM raster files and create a Canopy Height Model.
 > Then compare the two sites. Be sure to name your `R` objects and outputs
 > carefully, as follows: objectType_SJER (e.g. `DSM_SJER`). This will help you
 > keep track of data from different sites!
-> 
+>
 > 1. Import the DSM and DTM from the SJER directory (if not aready imported
-in the 
-> [Plot Raster Data in R]({{ site.baseurl }}/R/Plot-Rasters-In-R/) 
-tutorial.) Don't forget to examine the data for `CRS`, bad values, etc. 
+in the
+> [Plot Raster Data in R]({{ site.baseurl }}/R/Plot-Rasters-In-R/)
+tutorial.) Don't forget to examine the data for `CRS`, bad values, etc.
 > 2. Create a `CHM` from the two raster layers and check to make sure the data
-are what you expect. 
-> 3. Plot the `CHM` from SJER. 
+are what you expect.
+> 3. Plot the `CHM` from SJER.
 > 4. Export the SJER CHM as a `GeoTIFF`.
-> 5. Compare the vegetation structure of the Harvard Forest and San Joaquin 
-> Experimental Range. 
-> 
+> 5. Compare the vegetation structure of the Harvard Forest and San Joaquin
+> Experimental Range.
+>
 > Hint: plotting SJER and HARV data side-by-side is an effective way to compare
 both datasets!
 > > ## Answers
-> > ```{r challenge-code-SJER-CHM,include=TRUE, results="hide", echo=TRUE}
+> > ```{r challenge-code-SJER-CHM, include=TRUE, results="hide", echo=TRUE}
 > > # 1.
 > > # load the DTM
 > > DTM_SJER <- raster("data/NEON-DS-Airborne-Remote-Sensing/SJER/DTM/SJER_dtmCrop.tif")
 > > # load the DSM
 > > DSM_SJER <- raster("data/NEON-DS-Airborne-Remote-Sensing/SJER/DSM/SJER_dsmCrop.tif")
-> > 
+> >
 > > # check CRS, units, etc
 > > DTM_SJER
 > > DSM_SJER
-> > 
+> >
 > > # check values
-> > hist(DTM_SJER, 
+> > hist(DTM_SJER,
 > >      maxpixels=ncell(DTM_SJER),
 > >      main = "Digital Terrain Model - Histogram\n NEON SJER Field Site",
 > >      col = "slategrey",
 > >      ylab = "Number of Pixels",
 > >      xlab = "Elevation (m)")
-> > hist(DSM_SJER, 
+> > hist(DSM_SJER,
 > >      maxpixels=ncell(DSM_SJER),
 > >      main = "Digital Surface Model - Histogram\n NEON SJER Field Site",
 > >      col = "slategray2",
 > >      ylab = "Number of Pixels",
 > >      xlab = "Elevation (m)")
-> > 
+> >
 > > # 2.
 > > # use overlay to subtract the two rasters & create CHM
-> > CHM_SJER <- overlay(DSM_SJER,DTM_SJER,
+> > CHM_SJER <- overlay(DSM_SJER, DTM_SJER,
 > >                     fun=function(r1, r2){return(r1-r2)})
-> > 
```

Comment:
These challenge chunks need the extra line ending spaces to render properly.

Reply:
oh no problem! I didn't actually intend to remove a line! i did remove extra spaces at the end of a line... maybe that did it. i see you merged this. do you need a change here or did you just fix it @jsta

Reply:
I am in the process of fixing. No worries.

Reply:
ok great! sorry about that :) glad you caught it!